docs/01-start-here/06-testing-tips.md

```diff
@@ -6,11 +6,18 @@ and follow the steps to get your credentials for Frontend and for CMS-Fronts.
 This will save you grief later on.
 
 ### Install and compile all third party dependencies
-Compile your assets and third party dependencies using the command:
+To install third party dependencies and compile your assets as they would on production, use this command:
 ```
-  make install
+  make compile
 ```
-The make command can also be used for a variety of useful things incliding testing and linting javascript.
+If you wish to install dependencies and compile the assets in compile -dev mode, to allow real-time code compilation for testing of local branches, use the following:
+
+```
+make install
+make compile
```

Comment:
That is good too! But its pointless to use `make watch` if you are not writing SASS.